### DISCUSSION TITLE
[BUG] Error Logs Routine Required Survey: "Runtime Check Next Failed"

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#278] [4. Bug Reports] [answered]
Hola estoy intentando ejecutar el pb bot llamado PBTaskAddEditor en la rutina del required survey de un flujo de trabajo. 

Tiene la siguiente estructura mi pb bot:
````
invitar_gg: 
name: "PBTaskAddEditor"
version: "2.0.0"
data:
type: "user"
user: Array[1] ["60cb7354eec37600082f2f3c"]
task: "60cb83195fb95d79499ebc2c"
taskGroup: "60ca1c41b79c0f00086b72a0"
next:
SUCCESS: "message_sinlesion"
ERROR: ""
````

Y cuando ejecuto el bot llega a ese estado en el que me salta con el siguiente error: "Runtime Check Next Failed"

No se bien donde se encuentra el manejo de errores en el código de cotalker ): pero por el momento no he podido saber que cambios podría agregarle a mi rutina para evitar el error

Gracias!

**Comment by guillevalin** [comment, marked as answer]
@ValeMP hay un bug en la rutina PBTaskAddEditor. Se subirá una corrección a este incidente.
